core/capabilities/compute/compute.go

```diff
@@ -101,11 +102,17 @@ func (c *Compute) Execute(ctx context.Context, request capabilities.CapabilityRe
 		return capabilities.CapabilityResponse{}, fmt.Errorf("invalid request: %w", err)
 	}
 
+	var moduleCfg host.ModuleConfig
+	if err := yaml.Unmarshal(config, &moduleCfg); err != nil {
```

Comment:
@MStreet3 This isn't correct: `config` is the contents of the workflow config, it's not the capability config. You can find that req.Config and get what you need from it using UnwrapTo